docs/source/enterprise/cloud_media.rst

```diff
@@ -832,24 +832,24 @@ method to pass URLs for your S3-backed media when creating Labelbox data rows.
     any Google Cloud, Azure, or local media will still be uploaded to Labelbox
     as usual.
 
-.. _teams-cloud-functions:
+.. _enterprise-cloud-functions:
 
 AWS Lambda and Google Cloud Functions
 _____________________________________
 
-FiftyOne Teams can easily be used in AWS Lambda Functions and Google Cloud
+FiftyOne Enterprise can easily be used in AWS Lambda Functions and Google Cloud
 Functions.
 
 **Requirements**
 
-We recommend including Teams in your  function’s `requirements.txt` file by
-passing your token as a build environment variable, e.g.,
-`FIFTYONE_TEAMS_TOKEN` and then using the syntax below to specify the version
-of the Teams client to use:
+We recommend including FiftyOne Enterprise in your  function’s `requirements.txt`
+file by passing your token as a build environment variable, e.g.,
+`FIFTYONE_ENTERPRISE_TOKEN` and then using the syntax below to specify the version
```

Comment:
Are we changing env vars?

Reply:
no but this env var is localized to this example and doesn't correspond to any real env var